ParkingZoneApp/Areas/Admin/Controllers/ParkingSlotController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using ParkingZoneApp.Services;
+using ParkingZoneApp.Services.ParkingSlotService;
+using ParkingZoneApp.ViewModels.ParkingSlotViewModels;
+
+namespace ParkingZoneApp.Areas.Admin.Controllers;
+
+[Authorize]
+[Area("Admin")]
+public class ParkingSlotController : Controller
+{
+    private readonly IParkingSlotService _slotService;
+    private readonly IParkingZoneService _zoneService;
+
+    public ParkingSlotController(IParkingSlotService _slot, IParkingZoneService _zone)
+    {
+        _slotService = _slot;
+        _zoneService = _zone;
+    }
+
+    #region Index
+    // GET: Admin/ParkingSlot
+    public IActionResult Index(int id)
```

Comment:
It is better if you make id => ParkingZoneId because we will be confused about whether it is Id of Slot or ParkingZone